wicker/core/column_files.py

```diff
@@ -178,55 +183,110 @@ def _write_column(self, column_name: str) -> None:
         self.write_buffers[column_name] = self._get_new_buffer()
 
 
-class ColumnBytesFileCache:
+class ColumnBytesFileReader(abc.ABC):
+    """A reader class for column bytes files to inherit upon for caches and cacheless"""
+
+    def __init__(
+        self,
+        column_bytes_root_path: str,
+        dataset_name: Optional[str] = None,
+        path_factory: WickerPathFactory = S3PathFactory(),
+    ) -> None:
+        super().__init__()
+        self._column_bytes_root_path = column_bytes_root_path
+        self._datasent_name = dataset_name
```

Comment:
@pickles-bread-and-butter I caught a bug here: `self._datasent_name` -> `self._dataset_name`

Reply:
Lol, I had fixed this and forgot to push it up 🤣